### ISSUE TITLE
Upgrade 8.1->8.2 fails with: Table 'owncloud.oc_file_locks' doesn't exist

### ISSUE THREAD

**Issue by wscheicher** [owncloud/core#20111]
Upgraded Owncloud via Debian Packages from the Open Build Service.
Upgrade path was 7.0.10-1.1 ->  8.0.8-1.1 -> 8.1.3-2.1 -> 8.2.0-2.1 with sucessfully upgrade each step and login in the web interface to verify.

However, the last step ( 8.1.3-2.1 -> 8.2.0-2.1 ) failed with the web page permanently mentioning maintainence ( while top and ps aux didn't show anything actually being done )

So i tried to trigger the upgrade manually:

```
~root@hostname www/owncloud # ./occ upgrade
Console has to be executed with the user that owns the file config/config.php
Current user: root
Owner of config.php: www-data
PHP Fatal error:  Uncaught exception 'PDOException' with message 'SQLSTATE[42S02]: Base table or view not found: 1146 Table 'owncloud.oc_file_locks' doesn't exist' in /var/www/owncloud/3rdparty/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOStatement.php:91
Stack trace:
#0 /var/www/owncloud/3rdparty/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOStatement.php(91): PDOStatement->execute(Array)
#1 /var/www/owncloud/3rdparty/doctrine/dbal/lib/Doctrine/DBAL/Connection.php(989): Doctrine\DBAL\Driver\PDOStatement->execute(Array)
#2 /var/www/owncloud/lib/private/db/connection.php(203): Doctrine\DBAL\Connection->executeUpdate('DELETE FROM `oc...', Array, Array)
#3 /var/www/owncloud/lib/private/lock/dblockingprovider.php(184): OC\DB\Connection->executeUpdate('DELETE FROM `*P...', Array)
#4 /var/www/owncloud/lib/private/lock/dblockingprovider.php(189): OC\Lock\DBLockingProvider->cleanEmptyLocks()
#5 [internal function]: OC\Lock\DBLockingProvider->__destruct()
#6 {main}

Next exception 'Doctrine\DBAL\Driver\PDOException' with message 'SQLSTATE[42S in /var/www/owncloud/3rdparty/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php on line 53
```

On IRC i've been told to look at bug #19695 , however the fix mentioned there ( #19644 ) is in the package i used but still produced the above error.

**Comment by DeepDiver1975** [comment]
This is an issue we did fix for 8.2.1.

Workaround for now:
1. move your instance out of maintenance mode
2. rerun the upgrade

**Comment by wscheicher** [comment]
Ah, ok - thanks.
And the Workaround worked.
